python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -543,6 +540,26 @@ export default function EditorContainer({
     [deletePromptCallback, dispatch]
   );
 
+  const clearOutputsCallback = callbacks.clearOutputs;
+  const onClearOutputs = useCallback(
+    async () => {
+      try {
+        dispatch({
+          type: "CLEAR_OUTPUTS",
+        });
+        await clearOutputsCallback();
+      } catch (err: unknown) {
+        const message = (err as RequestCallbackError).message ?? null;
+        showNotification({
+          title: "Error clearing outputs",
+          message,
+          color: "red",
+        });
```

Comment:
Add a TODO to handle error boundaries, similar to what Ryan did in https://github.com/lastmile-ai/aiconfig/pull/799

Reply:
Error boundaries should only be used for catching rendering errors. Catching request exception should be done as is done here with try/catch and showing notification